### PR TITLE
chore: remaining Storybook components

### PR DESCRIPTION
**Pull request by jessicamcinchak**
Last of the outstanding ones per Trello checklist: 
- [x] Confirmation
- [x] FileUpload
- [x] FileUploadAndLabel
- [x] Result
- [x] Review
- [x] Section
- [x] Send
- [x] SetValue
- [x] TaskList
- [ ] ExternalPortal *
- [ ] Filter *
- [ ] InternalPortal *

`*` = Thesecomponent types DO NOT create a breadcrumb, therefore they have zero Public representation - which is different than "hidden" component types like Calculate or SetValue that still set a breadcrumb. I think I'd vote these are totally omitted from Storybook? I attemped to mock ExternalPortal & Filter but they look like a bit like misfits in my opinion.

### CODE REVIEW

**Comment by jessicamcinchak** [comment]
[Update Storybook](https://trello.com/c/xDSyYBbJ/2290-update-storybook)

File: editor.planx.uk/src/@planx/components/Review/Review.stories.tsx [hunk 1]
```diff
@@ -1,276 +1,54 @@
...
-    },
-    type: 200,
-  },
-  wViIvfuIJC: {
-    data: {
-      text: "(No Result)",
-    },
-    type: 200,
-  },
+// TODO improve mock data to include one component of each type
```
Comment (DafyddLlyr) [inline editor.planx.uk/src/@planx/components/Review/Review.stories.tsx:26]: We should also add  this step to the "adding a component" doc I guess?

File: editor.planx.uk/src/@planx/components/SetValue/SetValue.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+export const WithEditor = () => {
+  return (
+    <>
+      <Wrapper Editor={Editor} Public={Public} />
+      <WarningContainer>
+        <ErrorOutline />
+        <Typography variant="body2" ml={2}>
+          This component is only available in the Editor when designing
+          services, it does <strong>not</strong> display in the Public form.
```
Comment (DafyddLlyr) [inline editor.planx.uk/src/@planx/components/SetValue/SetValue.stories.tsx:24]: I think this is a good approach - super clear.

File: editor.planx.uk/src/@planx/components/Section/Section.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+const meta = {
+  title: "PlanX Components/Section",
+  component: Public,
+} satisfies Meta<typeof Public>;
+
+type Story = StoryObj<typeof meta>;
+
+export default meta;
+
+// TODO needs internal state
```
Comment (DafyddLlyr) [inline editor.planx.uk/src/@planx/components/Section/Section.stories.tsx:17]: I'd be happy to pick this one up!

**Review by DafyddLlyr — APPROVED** [review]
Thanks for this! 

I agree that the outstanding components don't really make much sense - we could have an Editor modal only displayed but there's not much there really to display.

File: editor.planx.uk/src/@planx/components/Review/Review.stories.tsx (hunk 1 above)
Comment (jessicamcinchak) [reply editor.planx.uk/src/@planx/components/Review/Review.stories.tsx:26]: Maybe, but it's really just one component _if_ shown on the Review page - not sure it's a meaningful step yet at that phase (still interested in thinking through less manual/duplicated mock data etc?) but will keep this one on the radar :ok_hand:

**Pull request merged**